src/MSLGenerator.h

```diff
@@ -88,10 +88,6 @@ class MSLGenerator
     Target          m_target;
     
     bool            m_error;
-    
-    // TODO: Temporary hack to access uniforms struct members
-    HLSLBuffer* m_per_pass_buffer;
```

Comment:
The reason for this change is that we obviously don't use per_item/per_pass names :) And we use up to 4 constant buffers in some shaders - so I made the code generic. I believe in this case it actually preserves behavior - I could run some shaders that do use per_item/per_pass cbuffer names and confirm this if you want.